spec/player_spec.rb

```diff
@@ -4,6 +4,7 @@
 describe Scrabble::Player do
   before :each do
     @bob = Scrabble::Player.new("Bob")
+    @tilebag = Scrabble::TileBag.new
```

Comment:
It seems like this `@tilebag` instance variable was unnecessarily added for all of your existing tests. Think about the fact that this variable would then be created for each of your existing tests where you aren't using it. I would recommend having another section of your RSpec tests that initialized this variable for only the tests where it is needed.